src/Microsoft.IdentityModel.Tokens/EncryptingCredentials.cs

```diff
@@ -109,6 +109,11 @@ public string Enc
             private set => _enc = string.IsNullOrEmpty(value) ? throw LogHelper.LogArgumentNullException("enc") : value;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public JsonWebKey JsonWebKey { get; set; }
```

Comment:
For ECDH, this can be a JsonWebKey OR a ECDsaSecurityKey.
Suggest a property

```cs
public SecurityKey KeyExchangePublicKey {get;set;}
```

Reply:
Changed